src/test/java/teammates/test/cases/search/FeedbackResponseCommentSearchTest.java

```diff
@@ -0,0 +1,97 @@
+package teammates.test.cases.search;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+import org.testng.annotations.Test;
+
+import teammates.common.datatransfer.FeedbackResponseCommentSearchResultBundle;
+import teammates.common.datatransfer.attributes.FeedbackResponseCommentAttributes;
+import teammates.common.datatransfer.attributes.InstructorAttributes;
+import teammates.storage.api.FeedbackResponseCommentsDb;
+
+/**
+ * SUT: {@link FeedbackResponseCommentsDb},
+ *      {@link teammates.storage.search.FeedbackResponseCommentDocument},
+ *      {@link teammates.storage.search.FeedbackResponseCommentSearchQuery}.
+ */
+public class FeedbackResponseCommentSearchTest extends BaseSearchTest {
```

Comment:
Yes, UI tests should not be used for unit testing back end logic. As long as there is a UI test to verify the front end is correctly wired to the backend and the frond end itself (alerts etc.) is working, that should be enough. For example, there should be at least one UI test that has this check box ticked to verify the check box is correctly wired up.
![image](https://user-images.githubusercontent.com/1673303/27623626-d71f18ba-5c0f-11e7-837e-6a3edf8cdade.png)